### DISCUSSION TITLE
Mediation

### DISCUSSION THREAD

**Discussion by Winamp9** [EMI-INDO/emi-indo-cordova-plugin-admob#28] [Documentation]
Hello everyone. I haven't tested mediation yet, but after installing the mediation plugins, I didn't notice the SDKs of the specified networks being added. Are the SDKs definitely added, or are they not required for these specific networks?

**Comment by EMI-INDO** [comment]
File a problem here https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/issues/new/choose
make sure to follow the template provided.
